wordpress-mobile/WordPress-iOS
# Fix UI tests and remove "Publish with confidence" dialog

Fixes #13855 and #13852

This PR fixes the UI tests and removes the "Publish with confidence dialog"

### To test (UI Tests)

I triggered UI tests for this PR. They're all green. :)

### To test (Publish with confidence)

1. Do a fresh install of WPiOS
2. Create a new post, tap "Publish..."
3. Check that no "Publish with confidence dialog" appears
